### PR TITLE
Gen3 setup scripts

### PR DESCRIPTION
**Pull request by dvenckusuchgo**
Move the dev setup scripts from configuration-files:/tools to compose-services:/tools

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+If you have stale Quay.io Gen3 docker images, you may want to destroy those and start fresh
+
+```
+# Destroy all Gen 3 Quay.io images
+docker images -a | grep quay | awk '{print $3}' | xargs docker rmi
+```
+
+### GEN3 SETUP
+
+In configuration-files:
```
Comment (grugna) [inline tools/README.md:45]: Does this still work with configuration-file or this is just from the old version?

File: tools/dev_setup.sh [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+#------------------------------------------------------
+echo "Running: creds_setup.sh"
+bash ./creds_setup.sh
+
+#-------------------------------------------------------
+# Copy Secrets, Configuration-Files --> Compose-Services
+#-------------------------------------------------------
+echo "Copying: configuration-files specialty items into compose-services"
+# sync
+cp -fr $CONFIG_FILES_DIR/compose-service/Secrets $COMPOSE_SVCS_DIR/
```
Comment (grugna) [inline tools/dev_setup.sh:93]: Are we assuming the user has already cloned the configuration-file repo in the CONFIG_FILES_DIR path?

File: tools/README.md (hunk 1 above)
Comment (dvenckusuchgo) [reply tools/README.md:45]: Needs to be corrected.  I will commit the change

File: tools/dev_setup.sh (hunk 2 above)
Comment (dvenckusuchgo) [reply tools/dev_setup.sh:93]: Needs to be handled now that we've relocated the location of the tools dir into compose-services

File: tools/README.md (hunk 1 above)
Comment (dvenckusuchgo) [reply tools/README.md]: Fixed.

File: tools/dev_setup.sh (hunk 2 above)
Comment (dvenckusuchgo) [reply tools/dev_setup.sh:111]: Added a check for configuration-files repo.  If it doesn't exist, will be cloned.

Fixed and Tested.

**Comment by grugna** [comment]
nice, you can merge. Thank you for the changes.

**Pull request merged**